plugin-directorymenu/directorymenu.cpp

```diff
@@ -107,10 +112,10 @@ void DirectoryMenu::openDirectory(const QString& path)
 
 void DirectoryMenu::openInTerminal(const QString& path)
 {
-    QProcess *process = new QProcess(this);
-    QString action = mDefaultTerminal + " --workdir " + path;
-    process->start(action);
-    delete process;
+    QProcess *thisProc = new QProcess(this);
+    QString action = mDefaultTerminal + " -w " + path;
+    mTermRefs.append(thisProc);
+    thisProc->start(action);
```

Comment:
IMO you just need to
```c++
connect(process, &QProcess::finished, process, &QObject::deleteLater);
```
before `start()` and let the pointer "leak", it will tidy up himself.

Or the second posibillity is to use `QProcess::startDetached()` (in this case the child process will not get any SIGHUP signal when the parent (and its terminal) is closed).